Engine/source/T3D/tsStatic.cpp

```diff
@@ -494,6 +498,21 @@ void TSStatic::prepRenderImage( SceneRenderState* state )
 {
    if( !mShapeInstance )
       return;
+//<!-- Scene Culling --!>
+   GameConnection* connection = GameConnection::getConnectionToServer();
```

Comment:
this is where it did the pre-render of the models in the scene to get rid of the hitch